.idea/checkstyle-idea.xml

```diff
@@ -16,4 +16,4 @@
       </list>
     </option>
   </component>
-</project>
+</project>
```

Comment:
Is this change included deliberately?

Reply:
Not really. Android studio had changed the file automatically. I just reverted it...